### DISCUSSION TITLE
Welcome to CLG Discussions!

### DISCUSSION THREAD

**Discussion by Gator7778** [Gator7778/CGESO#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rapaz-Jub** [comment]
![20250622212509_1](https://github.com/user-attachments/assets/d3d49e0d-51a4-4950-b391-e7a7c022a1c4)

**Comment by TwoToad** [comment]
Glenumbra Guide: Missing step to pick up the Vital Inheritance quest from NPC Alexia Dencent. Guide skips right to picking up Edrien's pipe.

--Twotoad, former contributor to the WoW-Pro Leveling guides.

**Reply by Gator7778** [reply]
going through all the old guides now trying to find all errors and glitches and trying to fix them. I havent found any bugs yet have found a few missing steps which im fixing along my way thank you for the post over here.
